api/desecapi/views.py

```diff
@@ -342,20 +340,20 @@ def create(self, request, *args, **kwargs):
             raise exc
 
     def perform_create(self, serializer):
-        serializer.save(domain=self.request.user.domains.get(name=self.kwargs['name']))
+        serializer.save(domain=self.domain)
 
     def update(self, request, *args, **kwargs):
+        partial = kwargs.pop('partial', False)
```

Comment:
Why is the default `False`? (Under which circumstances is this ever called without the `partial` argument?)

Reply:
that's DRF's default implementation, I just copied it 🙈